.github/workflows/downstream.yml

```diff
@@ -0,0 +1,56 @@
+name: Downstream
```

Comment:
I'd suggest changing it to a [composite action](https://docs.github.com/en/actions/creating-actions/creating-a-composite-action#creating-an-action-metadata-file) that would give it more flexibility